packages/edit-post/src/index.js

```diff
@@ -68,6 +68,15 @@ export function initializeEditor( id, postType, postId, settings, initialEdits )
 
 	registerCoreBlocks();
 
+	// Show a console log warning if the browser is not in Standards rendering mode.
+	try {
```

Comment:
Any particular reason for the `try/catch`?

Reply:
Old habit, I suppose, for any variable that I worry won't be supported by a particular browser.

Reply:
I don't really see anything that would break in this code, so I'd just leave them out if possible.